include/rviz_visual_tools/rviz_visual_tools.hpp

```diff
@@ -517,6 +517,19 @@ class RvizVisualTools
   bool publishABCDPlane(const double A, const double B, const double C, const double D,
                         Colors color = TRANSLUCENT, double x_width = 1.0, double y_width = 1.0);
 
+  /**
+   * \brief Display a plane given a vector normal to the plane and the distance to the plane along
+   *        that normal. The plane normal does not need to given as a unit vector.
+   * \param normal - a vector representing the normal of the plane
+   * \param d - the distance to the plane along the vector
+   * \param color - the color of the plane
+   * \param x_width - X-size of the vizualized plane [meters]
+   * \param y_width - Y-size of the visualized plane [meters]
+   * \return true on success
+   */
+  bool publishNormalAndDistancePlane(Eigen::Vector3d normal, double d, Colors color = TRANSLUCENT,
```

Comment:
```suggestion
  bool publishNormalAndDistancePlane(const Eigen::Vector3d& normal, const double d, const Colors color = TRANSLUCENT,
```

Reply:
`d` can't be const as I modify it in the function body, but good point on the other two